client/components/navbar/navbar-notification.service.js

```diff
@@ -21,33 +21,31 @@ class NavbarNotificationService {
   getSubLinks() {
     return this.getMessages()
       .then(messages => messages.map(message => this.convertSubLink(message)))
-      .catch(() => undefined);
+      .catch(() => this.$q.when([]));
   }
 
-  /* eslint-disable */
-  _formatTime(dateTime) {
+  static _formatTime(dateTime) {
     return moment(dateTime).fromNow();
   }
 
   _toggleSublinkAction(toUpdate, linkClicked) {
     if (toUpdate.isActive && !toUpdate.updating) {
-      toUpdate.updating = true;
+      _.set(toUpdate, 'updating', true);
```

Comment:
this was not part of the PR, it's code that have been already been reviewed.

Reply:
Related to a required rebase. (see: https://github.com/ovh-ux/ovh-manager-telecom/pull/741#discussion_r199218411)